Forge/src/main/java/me/ferdz/placeableitems/network/CNotifySpecialItemPlaceKeyPacket.java

```diff
@@ -0,0 +1,51 @@
+package me.ferdz.placeableitems.network;
+
+import net.minecraft.network.PacketBuffer;
+
+/**
+ * A client -> server packet to inform it of a client-sided keybind update for the
+ * special item place key.
+ *
+ * @author Parker Hawke - Choco
+ */
+public class CNotifySpecialItemPlaceKeyPacket {
```

Comment:
Should be renamed to `CNotifyItemPlaceKeyPacket `

Reply:
Forgot about the packet. Whoops. Will rename that :)